frontstage/views/passwords.py

```diff
@@ -24,6 +25,46 @@ def forgot_password():
         email_address = request.form.get('email_address')
         post_data = {"email_address": email_address}
 
+        try:
+            token_url = app.config['ONS_TOKEN']
+
+            data = {
+                'grant_type': 'reset_password',
+                'client_id': app.config['RAS_FRONTSTAGE_CLIENT_ID'],
+                'client_secret': app.config['RAS_FRONTSTAGE_CLIENT_SECRET'],
+                'username': email_address,
+            }
+            headers = {
+                'Accept': 'application/json',
+                'Content-Type': 'application/x-www-form-urlencoded;charset=UTF-8',
```

Comment:
Should content-type be `application/json`?

Reply:
I haven't tested it but if I remember right the Django server is a bit picky about what you put as content type and this was the only way we could get it working.
I'm working on a similar call in frontstage-api so I can test

Reply:
No the Django oAuth server is expecting an encoded payload, breaks if i set content-type to application/json